src/helpers/childrenAsArray.ts

```diff
@@ -0,0 +1,11 @@
+import { ReactNode } from 'react';
+
+export default function childrenAsArray(children: ReactNode): ReactNode[] {
+  if (Array.isArray(children)) {
+    return children;
+  } else if (typeof children !== 'undefined') {
```

Comment:
I don't like `else-if` statement, and you don't need a simple `if` would do the job